mozilla_vpn/explores/usage_mau.explore.lkml

```diff
@@ -0,0 +1,5 @@
+include: "../views/usage_mau.view"
+
+explore: usage_mau {
+  description: "VPN Monthly Active Users (based on Glean telemetry)"
```

Comment:
You can also give it a custom label (MAU, WAU, DAU might not be clear to everyone maybe). The label is used in the "Explores" list.
```suggestion
  description: "VPN Monthly Active Users (based on Glean telemetry)"
  label: "Monthly Usage"
```